src/main/java/uk/ac/ebi/eva/submission/controller/SubmissionController.java

```diff
@@ -0,0 +1,30 @@
+package uk.ac.ebi.eva.submission.controller;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.eva.submission.model.SubmissionStatus;
+import uk.ac.ebi.eva.submission.service.SubmissionService;
+
+@RestController
+@RequestMapping("/v1")
+public class SubmissionController {
+    private SubmissionService submissionService;
+
+    public SubmissionController(SubmissionService submissionService) {
+        this.submissionService = submissionService;
+    }
+
+    @PostMapping("ftp")
+    public String createFtpDirectory() {
+        SubmissionStatus submissionStatus = submissionService.createFtpDirectory();
+        return submissionStatus.getSubmissionId();
+    }
+
+    @PutMapping("completed")
+    public String markSubmissionComplete(@RequestBody String submissionId) {
+        return submissionService.markSubmissionComplete(submissionId).toString();
+    }
```

Comment:
I think we might want to discuss the names of the FTP endpoints.
I was thinking it should be something that reflect the process of submission rather than underlying action performed by the API. 
I also think it needs to makes sense with the HTTP request type. We can have a discussion about this in subsequence DEV meeting.

Reply:
updated end-points name